src/frontend/next/src/pages/index.tsx

```diff
@@ -1,4 +1,5 @@
 import Head from 'next/head';
+import Footer from '../components/Footer/index';
```

Comment:
Change this to `import Footer from '../components/Footer';` and the styling should be applied properly

Reply:
Fixed, thanks!